final_prep/README.md

```diff
@@ -5,8 +5,8 @@ Congrats on making it to the Mod 0 Final Prep! Complete the final exercises belo
 
 You've learned a ton about some of the core foundations of Javascript! Show us how far you've come by completing the following exercises! You will be using your work from these exercises in your first day of Mod 1! 
 
-- [ ] Complete the [Mod Zero Hero Challenge](./mod_zero_hero.rb)
-- [ ] Complete the [Annotation Challenge](./annotations.rb)
+- [x] Complete the [Mod Zero Hero Challenge](./mod_zero_hero.rb)
+- [x] Complete the [Annotation Challenge](./annotations.rb)
```

Comment:
I do not see any changes for your `final_prep/annotations.rb` or `final_prep/mod_zero_hero.rb`.

Reply:
Hello Ellen, I sent you a message in slack as well. I finished the work on time I just forgot to push it. I hope this is not an issue. I have pushed it through though, let me know if you have any other questions.